src/gui/FileBrowser.cpp

```diff
@@ -189,10 +189,10 @@ void FileBrowser::addContentCheckBox()
 }
 
 void FileBrowser::saveDirectoriesStates()
-{	
```

Comment:
I appreciate you cleaning up the spaces but generally, we are against out of scope whitespace changes. You can add in the spaces again and the diff will cancel out. Clean up only those lines you work on.

Reply:
Done